docs/Hyprland-Lua-Rollout.md

```
@@ -0,0 +1,109 @@
+login. That file is not HorneroConfig's legacy configuration; `hyprland.conf.d/`
+is removed by this migration, and a fresh Hyprland start with `hyprland.lua`
+present will use the Lua entrypoint.
+
+Or, for an immediate live-session fallback without changing git state:
+
+```sh
+mv ~/.config/hypr/hyprland.lua ~/.config/hypr/hyprland.lua.bak
+hyprctl reload
+```
+
+3. If all else fails and you need an immediate live-session fallback, disable
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)